src/webgpu/api/validation/createPipelineLayout.spec.ts

```diff
@@ -85,8 +85,12 @@ g.test('visibility_and_dynamic_offsets')
     const { type, hasDynamicOffset, visibility } = t.params;
     const info = kBindingTypeInfo[type as GPUBindingType];
 
+    const storageTextureFormat =
+      type === 'readonly-storage-texture' || type === 'writeonly-storage-texture'
+        ? ('r32uint' as const)
+        : undefined;
```

Comment:
Should it be valid to always specify this, even for non-storage texture formats?

Reply:
I'm not sure. Currently the spec allows it. I'm iffy on whether we should, though, because it allows storageTextureFormat to be completely ignored (we don't check it at all, beyond bindings, if it's not a storage texture) so it could have any random texture format in it. I think we should keep the test like this.

Reply:
Filed https://github.com/gpuweb/gpuweb/issues/829